java/odsa-jdbc-atp/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.oracle.dev.jdbc</groupId>
+  <artifactId>odsa-jdbc-atp</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>odsa-jdbc-atp</name>
+  <description>A simple JDBC + ODSA ATP app example</description>
+  <url>
+    https://medium.com/oracledevs/odsa-for-java-developers-introduction-to-oracle-database-service-for-azure-part-1-bff24c787055</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <!-- Oracle JDBC JARs -->
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc11</artifactId>
+      <version>23.4.0.24.05</version>
+    </dependency>
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>oraclepki</artifactId>
+      <version>19.11.0.0</version>
```

Comment:
oraclepki should always be on the same version as the driver.

Reply:
Hi @jeandelavarene, I have already replied to your request related to that. I checked that and tried to align previously indeed. However, there's no way to do so, as the links below do not have it; this is the latest published version, indeed.

Have a look and confirm, otherwise, please provide instructions on how to enable it as it's not available as we can see:

*** We have 19.11.0.0 only ***

https://repo1.maven.org/maven2/com/oracle/database/jdbc/oraclepki/
https://mvnrepository.com/artifact/com.oracle.database.jdbc/oraclepki

Reply:
updated to align the driver and oraclepki versions, then use the latest release of both. @Kuassim please review, thanks

Reply:
Note that although nobody informed the right URL, I found it under https://mvnrepository.com/artifact/com.oracle.database.security/oraclepki and not https://mvnrepository.com/artifact/com.oracle.database.jdbc/oraclepki, so the versions are aligned now.